python/pyspark/ml/wrapper.py

```diff
@@ -144,7 +158,9 @@ def _transfer_params_from_java(self):
             if self._java_obj.hasParam(param.name):
                 java_param = self._java_obj.getParam(param.name)
                 # SPARK-14931: Only check set params back to avoid default params mismatch.
-                if self._java_obj.isSet(java_param):
+                if self._java_obj.isSet(java_param) or (
+                        # SPARK-10931: Temporary fix for params that have a default in Java
+                        self._java_obj.hasDefault(java_param) and not self.isDefined(param)):
```

Comment:
This change will make a default value for a param in java side as an user-provided param value in python side.